neo/Ledger/Blockchain.cs

```diff
@@ -283,7 +283,10 @@ private RelayResultReason OnNewBlock(Block block)
                     block_cache_unverified.Remove(blockToPersist.Index);
                     Persist(blockToPersist);
 
-                    if (blocksPersisted++ < blocksToPersistList.Count - (2 + Math.Max(0, (15 - SecondsPerBlock)))) continue;
+                    // 15000 is the default among of seconds per block, while MilliSecondsPerBlock is the current
+                    uint extraBlocks = (15000 - MillisecondsPerBlock) / 1000;
```

Comment:
15000 is a hardcoded value, you should take this from the config

Reply:
That is right, it has been like this. I just adjusted to ms.
We can do that in another PR, for avoiding other possible mistakes.

Furthermore, I am not good with this config until nowdays...

Reply:
This, should take the value from the configuration

Reply:
I don't understand this logic very well ... why it used this?

Reply:
As faster the network is (privatenets with 1s) more blocks are rebroadcasted each time you Persist, limitted to 2+15.